contrib/test_jenkins.sh

```diff
@@ -58,6 +58,81 @@ echo "Build without IB verbs"
 ../contrib/configure-release --without-verbs
 make $make_opt
 
+if [ -n "$JENKINS_PPC_SERVER" ]; then
```

Comment:
please try to unite this with the other case (JENKINS_RUN_TESTS), avoid code duplication